### ISSUE TITLE
Warehouse people list duplicates

### ISSUE THREAD

**Issue by johnvanbreda** [Indicia-Team/warehouse#227]
The warehouse people list page (/index.php/person) shows a list of all people records, with a filter on website_id to apply permissions for people who are not warehouse admin. This filter means that a website_id column needs to be included in the view output. Therefore if there are several websites linked to a person (via their user account) then the same person might appear multiple times in the list.

**Comment by JimBacon** [comment]
Not sure I understand. Is the issue that the people list is showing duplicates when it should not?

**Comment by johnvanbreda** [comment]
Yes, correct. It is a minor point as they all link to the same person, but you will get several copies of the same person appearing in the table. Ideally there should only be one.
